config/lib/seed.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+  chalk = require('chalk'),
+  crypto = require('crypto'),
+  User = mongoose.model('User');
+
+console.log(chalk.bold.red('Warning:  Database seeding is turned on'));
+
+//If production only seed admin if it does not exist
+if (process.env.NODE_ENV === 'production') {
```

Comment:
@trainerbill What's the purpose of this block? If the environment is *production* you're creating the Admin account, if it doesn't exist. But below at https://github.com/meanjs/mean/pull/785/files#diff-7dfe67f86c4c0773b98c0422a0cb606aR64, you're removing and re-creating the Admin account anyway. I'm just trying to understand what's going on there; not that it's wrong. Perhaps, add a bit more comments? 

Also, how about breaking this file up into exported funcitons? With an `init` routine, and a couple maintenance/utility functions that will allow you to interact with the seeded users from other area's of the application? For instance, you can have a function that removes/locks one of the accounts.